# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/mnemoc/vquota/vserver-0.22-cap_quotactl.diff
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./chcontext.cc.orig	Thu Aug  8 21:31:32 2002
+++ ./chcontext.cc	Tue Feb 11 04:05:58 2003
@@ -107,7 +107,8 @@
 		|(1<<CAP_SYS_NICE)
 		|(1<<CAP_SYS_RESOURCE)
 		|(1<<CAP_SYS_TIME)
-		|(1<<CAP_MKNOD);
+		|(1<<CAP_MKNOD)
+		|(1<<CAP_QUOTACTL);
 	const char *hostname=NULL, *domainname=NULL;
 
 	for (i=1; i<argc; i++){
@@ -178,6 +179,7 @@
 				{"CAP_SYS_RESOURCE",CAP_SYS_RESOURCE},
 				{"CAP_SYS_TIME",	CAP_SYS_TIME},
 				{"CAP_MKNOD",		CAP_MKNOD},
+				{"CAP_QUOTACTL",	CAP_QUOTACTL},
 				{NULL,0}
 			};
 			int j;
--- ./reducecap.cc.orig	Thu Jul  4 11:11:50 2002
+++ ./reducecap.cc	Tue Feb 11 04:05:58 2003
@@ -54,6 +54,7 @@
 		"CAP_MKNOD",
 		"CAP_LEASE",
 		"CAP_OPENDEV",
+		"CAP_QUOTACTL",
 		NULL
 	};
 	int i;
@@ -168,6 +169,7 @@
 				{"--SYS_RESOURCE",CAP_SYS_RESOURCE},
 				{"--SYS_TIME",	CAP_SYS_TIME},
 				{"--MKNOD",		CAP_MKNOD},
+				{"--QUOTACTL",		CAP_QUOTACTL},
 				{NULL,0}
 			};
 			int j;
